zone/gm_commands/npcedit.cpp

```diff
@@ -1,552 +1,847 @@
-		c->Message(Chat::White, "#npcedit lastname - Sets an NPC's Lastname");
-		c->Message(Chat::White, "#npcedit level - Sets an NPC's Level");
-		c->Message(Chat::White, "#npcedit race - Sets an NPC's  Race");
-		c->Message(Chat::White, "#npcedit class - Sets an NPC's Class");
-		c->Message(Chat::White, "#npcedit bodytype - Sets an NPC's Bodytype");
-		c->Message(Chat::White, "#npcedit hp - Sets an NPC's Hitpoints");
-		c->Message(Chat::White, "#npcedit mana - Sets an NPC's Mana");
-		c->Message(Chat::White, "#npcedit gender - Sets an NPC's Gender");
-		c->Message(Chat::White, "#npcedit texture - Sets an NPC's Texture");
-		c->Message(Chat::White, "#npcedit helmtexture - Sets an NPC's Helmet Texture");
-		c->Message(Chat::White, "#npcedit herosforgemodel - Sets an NPC's Hero's Forge Model");
-		c->Message(Chat::White, "#npcedit size - Sets an NPC's Size");
-		c->Message(Chat::White, "#npcedit hpregen - Sets an NPC's Hitpoints Regeneration Rate Per Tick");
-		c->Message(Chat::White, "#npcedit hp_regen_per_second - Sets an NPC's HP regeneration per second");
-		c->Message(Chat::White, "#npcedit manaregen - Sets an NPC's Mana Regeneration Rate Per Tick");
-		c->Message(Chat::White, "#npcedit loottable - Sets an NPC's Loottable ID");
-		c->Message(Chat::White, "#npcedit merchantid - Sets an NPC's Merchant ID");
-		c->Message(Chat::White, "#npcedit alt_currency_id - Sets an NPC's Alternate Currency ID");
-		c->Message(Chat::White, "#npcedit spell - Sets an NPC's Spells List ID");
-		c->Message(Chat::White, "#npcedit npc_spells_effects_id - Sets an NPC's Spell Effects ID");
-		c->Message(Chat::White, "#npcedit faction - Sets an NPC's Faction ID");
-		c->Message(Chat::White, "#npcedit adventure_template_id - Sets an NPC's Adventure Template ID");
-		c->Message(Chat::White, "#npcedit trap_template - Sets an NPC's Trap Template ID");
-		c->Message(Chat::White, "#npcedit damage [minimum] [maximum] - Sets an NPC's Damage");
-		c->Message(Chat::White, "#npcedit attackcount - Sets an NPC's Attack Count");
-		c->Message(Chat::White, "#npcedit special_attacks - Sets an NPC's Special Attacks");
-		c->Message(Chat::White, "#npcedit special_abilities - Sets an NPC's Special Abilities");
-		c->Message(Chat::White, "#npcedit aggroradius - Sets an NPC's Aggro Radius");
-		c->Message(Chat::White, "#npcedit assistradius - Sets an NPC's Assist Radius");
-		c->Message(Chat::White, "#npcedit featuresave - Saves an NPC's current facial features to the database");
-		c->Message(Chat::White, "#npcedit armortint_id - Sets an NPC's Armor Tint ID");
-		c->Message(Chat::White, "#npcedit color [red] [green] [blue] - Sets an NPC's Red, Green, and Blue armor tint");
-		c->Message(Chat::White, "#npcedit ammoidfile - Sets an NPC's Ammo ID File");
+		c->Message(Chat::White, "You must target an NPC to use this command.");
+		return;
+	}
+
+	int arguments = sep->argnum;
+
+	bool is_help = arguments >= 1 ? !strcasecmp(sep->arg[1], "help") : true;
```

Comment:
This may be more clear as something like
`bool is_help = (arguments == 0 || !strcasecmp(sep->arg[1], "help"))` instead of using a ternary